tokens/colors.json

```diff
@@ -0,0 +1,76 @@
+{
+  "global": {
+    "type": "color",
+    "category": "background-color"
+  },
+  "props": {
+    "COLOR_BRAND_WHITE": {
+      "value": "#FFFFFFF",
+      "comment": "Brand color white"
+    },
+    "COLOR_BRAND_GREY-LIGHT": {
```

Comment:
We are not consistent in useing `-` or `_`. Here we have `GREY-LIGHT`, and later we have `GREEN_LIGHT`. My preference would be to always snake case it.